.storybook/addon-a11yDocs/register.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { addons, types } from '@storybook/addons';
+import { useParameter } from '@storybook/api';
+import { Description, DocsPageWrapper } from "@storybook/components"
+import {styled, css} from '@storybook/theming';
+
+const A11yDocs = () => {
+  const value = useParameter('a11yDocs', null) || '';
+  return (
+    <DocsPageWrapper>
+      <Description markdown={value}></Description>
+    </DocsPageWrapper>    
+  );
+};
+
+addons.register('a11ydocs', () => {
+  addons.add('a11ydocs/tab', {
+    type: types.TAB,
+    title: 'Accessibility Docs',
```

Comment:
Maybe just "Accessibility"?

Reply:
We already have "Accessibility" panel, this can be confusing